src/goPackages.ts

```diff
@@ -149,12 +182,21 @@ export function getImportablePackages(filePath: string, useCache: boolean = fals
 				return;
 			}
 
-			let relativePkgPath = getRelativePackagePath(currentFileDirPath, currentWorkspace, pkgPath);
+			if (!foundPkgRootDir) {
```

Comment:
@uudashr 
You will have to find the `pkgRootDir` upfront and not after getting the results from `gopkgs`. To understand why, take the below case

- If you dont have the `godef` code, then run `go get github.com/rogpeppe/godef` 
- Open the `godef` folder and then open the file `ast.go` in it
- Run the command `Go: Add Import` with the current Go extension and then with your changes and search for `acme`
- In the current Go extension, you will get the result for "9fans.net/go/acme", but you wont get the same with your changes.

This is because, the `acme` package is in the vendor folder which is not a sibling of the directory to which the file `ast.go` file belongs.

Reply:
No, it should works.

Don't know which version `gopkgs` did you use. Since I push the latest changes, gopkgs `-workDir` also works on sub packages. Try to update the `gopkgs`

```
$ gopkgs -format '{{.Name}};{{.ImportPath}}' -workDir=$(go env GOPATH)/src/github.com/rogpeppe/godef/go/ast|grep acme
acme;github.com/rogpeppe/godef/vendor/9fans.net/go/acme
```

And this also covered on the test case *"Replace vendor packages with relative path"*

Reply:
Works in my Mac but not Windows

Reply:
Just push an update... would you mind to test it on Windows?